src/Trip.js

```diff
@@ -0,0 +1,79 @@
+class Trip {
+  constructor({
+    id,
+    userID,
+    destinationID,
+    travelers,
+    date,
+    duration,
+    status,
+    suggestedActivities
+  }, destinations) {
+    this.id = id;
+    this.userID = userID;
+    this.destination = destinations.filter(destination => destination.id === destinationID)[0]
+    this.travelers = travelers;
+    this.date = date;
+    this.duration = duration;
+    this.status = status;
+    this.suggestedActivities = suggestedActivities;
+  }
+
+  calculateTripCost() {
+    const flightCost = this.travelers * this.destination.estimatedFlightCostPerPerson;
+    const lodgingCost = this.duration * this.destination.estimatedLodgingCostPerDay;
+    const expenses = flightCost + lodgingCost;
+    const agentFee = expenses * .1;
+    return expenses + agentFee
```

Comment:
I like that you included the agentFee in this method and didn't break it out into its own function. Nice work!